app/src/main/java/com/gh4a/adapter/SearchAdapter.java

```diff
@@ -127,6 +127,7 @@ public void onBindViewHolder(ViewHolder holder, SearchCode result) {
                     tvMatch.setText(builder);
                     tvMatch.setTag(result);
                     tvMatch.setTag(R.id.search_match_index, i);
+                    tvMatch.setTypeface(Typeface.MONOSPACE);
```

Comment:
Can this be done in XML?

Reply:
Unfortunately no, because at the moment StyleableTextView doesn't handle the `typeface` attribute correctly. (#1166 will fix that)